src/Base/Body.php

```diff
@@ -165,6 +165,30 @@ protected function matchArray($name, $schema, $body, $type)
         return true;
     }
 
+    /**
+     * Checks if the value is valid date.
+     *
+     * @param $name
+     * @param $schema
+     * @param $body
+     * @param $type
```

Comment:
Types?

Reply:
I did not want to change the "code" style (I've copied the) same structure as matchString and etc. I can of course add correct types.